tests/unit/test_duckdb.py

```diff
@@ -56,6 +56,25 @@ def test_create_table_from_parquet_append(duckdb, TEST_PARQUET_FILE_PATH):
     duckdb.run(f"DROP SCHEMA {SCHEMA}")
 
 
+def test_create_table_from_parquet_delete(duckdb, TEST_PARQUET_FILE_PATH):
+    duckdb.create_table_from_parquet(
+        schema=SCHEMA, table=TABLE, path=TEST_PARQUET_FILE_PATH
+    )
+    df = duckdb.to_df(f"SELECT * FROM {SCHEMA}.{TABLE}")
+    assert df.shape[0] == 3
+
+    # if exists = 'delete' deletes data and then inserts new one
+
+    duckdb.create_table_from_parquet(
+        schema=SCHEMA, table=TABLE, path=TEST_PARQUET_FILE_PATH, if_exists="delete"
+    )
+    df = duckdb.to_df(f"SELECT * FROM {SCHEMA}.{TABLE}")
+    assert df.shape[0] == 3
+
+    duckdb.drop_table(TABLE, schema=SCHEMA)
+    duckdb.run(f"DROP SCHEMA {SCHEMA}")
+
+
```

Comment:
One way to test delete specifically is to load some data with a new schema and make sure this load fails.

Reply:
Once you resolve this one, pls resolve the conflicts as well